context/AuthContextProvider.tsx

```diff
@@ -32,16 +37,29 @@ export const AuthContextProvider = ({
         return;
       }
 
+      return setIsSignedIn(true);
```

Comment:
Returning a setState?

Reply:
yup! that was a boo boo and a brain fart.  Although it didn't break the app I don't think it is right.  I'll fix it!

Reply:
✅ Done!